104-max-depth-binary-tree.js

```diff
@@ -0,0 +1,49 @@
+// Challenge: https://leetcode.com/problems/maximum-depth-of-binary-tree/
+
+// Objective: Given the root of a binary tree, return its maximum depth.
+
+// Note: A binary tree's maximum depth is the number of nodes along the longest path
+// from the root node down to the farthest leaf node.
+
+// Solution:
+const assert = require("assert");
+
+/*
+ Definition for a binary tree node.
+ function TreeNode(val, left, right) {
+ 	this.val = (val===undefined ? 0 : val)
+ 	this.left = (left===undefined ? null : left)
+  this.right = (right===undefined ? null : right)
+  }
+*/
+/*
+ @param {TreeNode} root
+ @return {number}
+*/
+
+var maxDepth = function(root) {
```

Comment:
no changes needed but prefer `const` over `var`. If reassignment is necessary, prefer `let`